apps/e2e/src/indexer-client/fullSanity.ts

```diff
@@ -138,7 +139,7 @@ async function fullSanity(context: RunContext) {
 
   const now = nowInSeconds();
   const multiTimestampProductSnapshots = await client.getMultiProductSnapshots({
-    productIds: [1, 2, 3],
+    productIds: [0, 2, 4],
```

Comment:
This was needed to make the sanity test work on blast testnet (product ID of 1 doesn't exist)